app/components/holidays-overview.js

```diff
@@ -0,0 +1,57 @@
+import Component from '@glimmer/component';
+import Duration from '../utils/duration';
+import { trackedRef } from 'ember-ref-bucket';
+import { offset } from '@floating-ui/dom';
+import { trackedFunction } from 'reactiveweb/function';
+import { keepLatest } from 'reactiveweb/keep-latest';
+import { service } from '@ember/service';
+import { startOfYear } from 'date-fns';
+import { formatDate } from 'frontend-timekeeper/utils/format-date';
+import { use } from 'ember-resources';
+import constants from 'frontend-timekeeper/constants';
+const { HOLIDAY_TASK_LABEL } = constants;
```

Comment:
Is undefined. Must be `HOLIDAY_TASK_NAME` (or maybe better to change the name in `constants.js`)